src/pages/documentation/turn/connecting.astro

```diff
@@ -19,9 +19,11 @@ const page: IStandardPageMetadata = {
   pageDescription: 'Information on how to get connected to the TURN and Nevada GPS networks.',
```

Comment:
nitpick: on the previous page (/documentation/turn/) we refer to it as the Reno network. Ignore this remark if they are interchangeable.

Reply:
we should be consistent and choose one. Since it's only recommended for Reno users, should it be called the Reno network?

Reply:
It was decided to use Nevada GPS and TURN GPS everywhere as the product name. We'll have to find all occurences and titles and update them.